classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,323 @@
+	}
+	public void methodEnn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodFn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodGn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodHn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodIn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodLn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnAn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnBn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnCn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnDn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnEn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnFn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnGn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnHn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBnIn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBLn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCAn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnBn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnCn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnDn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnEn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnFn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnGn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnHn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnIn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCnLn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnAn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnCn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnDn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnEn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnFn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnGn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnHn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnIn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDnLn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEnAn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEnBn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEnCn(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodEnDn(){
+		Integer i = 0;
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.